303-quarkus-vertx-sql/src/test/java/io/quarkus/qe/vertx/sql/dbpool/PostgresPoolTest.java

```diff
@@ -161,7 +161,7 @@ public void checkBorderConditionBetweenIdleAndGetConnection(){
                                     }
                                     return result;
                                 }).toMulti();
-                    }).collect().in(ArrayList::new, List::add).subscribe().with(re -> {
+                    }).collectItems().in(ArrayList::new, List::add).subscribe().with(re -> {
```

Comment:
What triggers `.collect() => collectItems()` ?

Reply:
It's not supported on 1.11.3. The method `collectItems` has been replaced by `collect()` on version 1.12

Reply:
This should be highlighted at least in https://github.com/quarkusio/quarkus/wiki/Migration-Guide-1.12
Changes of Uni / Multi related API can be trouble in the future, it should really stabilize.
Please talk to Clement about `collectItems() => collect()`  shift in Quarkus 1.12 and suggest to enhance the migration guide. Another option is GH issue about it, but I would start with ping on zulip